read-fonts/src/tables/postscript.rs

```diff
@@ -14,7 +16,9 @@ pub enum Error {
     /// The `off_size` field in an INDEX contained an invalid value.
     InvalidIndexOffsetSize(u8),
     /// An INDEX contained a zero offset.
-    ZeroOffset,
+    ZeroOffsetInIndex,
+    /// Variation store index referenced an invalid variation region.
+    InvalidVsIndex(u16),
```

Comment:
Suggest InvalidVariationStoreIndex in full; Vs is a bit obtuse.

Reply:
Done